includes/payments/actions.php

```diff
@@ -171,8 +171,7 @@ function edd_undo_purchase_on_refund( $payment_id, $new_status, $old_status ) {
 
 	if( $customer_id ) {
 
-		$customer = new EDD_Customer( $customer_id );
-		$customer->decrement_stats( $customer_id, $amount );
+		EDD()->customers->decrement_stats( $customer_id, $amount );
```

Comment:
That has changed, we don't use EDD->customer to modify a customer's data anymore. That was changed with #2548 & #3018  and all the relevant areas were updated, if you find one that isn't properly updated for updating a customer's stats, submit it against issue #2548 or #3018